motore/Cargo.toml

```diff
@@ -33,3 +33,7 @@ http = "0.2"
 
 [features]
 tower = ["dep:tower"]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
Please add a newline to end of file.

Reply:
already add it

Reply:
[/// This is similar to [`BoxService`](super::BoxService) except the resulting](/// This is similar to [`BoxService`](super::BoxService) except the resulting)

This line mention `BoxService`, but `BoxService` doesn't impl in super::BoxService.